src/components/views/rooms/wysiwyg_composer/DynamicImportWysiwygComposer.tsx

```diff
@@ -39,8 +39,24 @@ export const dynamicImportSendMessage = async (
 };
 
 export const dynamicImportConversionFunctions = async (): Promise<{
+    /**
+     * Creates a rust model from rich text input (html) and uses it to generate the plain text equivalent (which may
+     * contain markdown). The return value must be used to set `.innerHTML` for consistency in handling linebreaks.
+     *
+     * @param rich - html to convert
+     * @returns a string of plain text that may contain markdown
+     */
     richToPlain(rich: string): Promise<string>;
-    plainToRich(plain: string): Promise<string>;
+
+    /**
+     * Creates a rust model from plain text input (interpreted as markdown) and uses it to generate the rich text
+     * equivalent. Output can be formatted for display in the composer or for sending in a Matrix message.
+     *
+     * @param plain - plain text to convert, read as innerHTML
```

Comment:
Let's expand on this a bit:

```suggestion
     * @param plain - plain text to convert. Note: when reading the plain text from the editor element, be sure to
     *     use `.innerHTML` (rather than `.innerText`) to ensure that punctuation characters are correctly HTML-encoded.
```

again, not 100% sure that is accurate?